controllers/storagecluster/cephcluster.go

```diff
@@ -94,6 +96,24 @@ func arbiterEnabled(sc *ocsv1.StorageCluster) bool {
 	return sc.Spec.Arbiter.Enable
 }
 
+func (r *StorageClusterReconciler) deleteDuplicatePlacementJobs(job batchv1.Job) error {
```

Comment:
We can rename this to something more appropriate like deletePlacementJobsWithDuplicateTolerations